### ISSUE TITLE
helm - Enabling Custom Resources causes name mismatch with global configurations file

### ISSUE THREAD

**Issue by gbarrett1988** [nginxinc/kubernetes-ingress#1160]
**Describe the bug**
Using helm chart v1.8.1. 

values.yaml
```
...
enableCustomResources: true
globalConfiguration:
  create: true
  spec:
    listeners:
      - name: ssh
         port: 22
         protocol: TCP
...
```

Just trying to setup an SFTP server in a cluster and use the existing nginx ingress controller. I know I need to create a transport server to do this but when I apply the values.yaml above to the controller it blows up with:

`Error when getting nginx/test: globalconfigurations.k8s.nginx.org "nginx" not found`

Output of 
```
kubectl get gc:
```
```
NAME                 AGE
nginx-nginx-ingress   16m
```

So clearly the controller pod is looking for a different name ("nginx") than what was created during the helm upgrade ("nginx-nginx-ingress").

**Your environment**
* Version of the Ingress Controller - release version or a specific commit: tag v1.8.1
* Version of Kubernetes: 1.19.1
* Kubernetes platform (e.g. Mini-kube or GCP): Azure
* Using NGINX or NGINX Plus: NGINX

**Additional context**
Add any other context about the problem here. Any log files you want to share.

**Comment by gbarrett1988** [comment]
Realizing now that this appears to be patched in master branch but not in Release v1.8.1.

**Comment by pleshakov** [comment]
Hi @gbarrett1988 

 For 1.8.1, please see a workaround https://github.com/nginxinc/kubernetes-ingress/issues/1103#issuecomment-677964409
